cms/djangoapps/contentstore/views/utilities/bulksettings.py

```diff
@@ -9,14 +9,113 @@
 
 from edxmako.shortcuts import render_to_response
 
-from contentstore.utils import BulkSettingsUtil
-
+from contentstore.views.helpers import xblock_studio_url
 from opaque_keys.edx.keys import CourseKey
 from xmodule.modulestore.django import modulestore
 
 from student.auth import has_course_author_access
 from ..course import get_course_and_check_access
 
+from datetime import datetime
```

Comment:
Sorry for the confusion; I must have gotten mixed up in the commits. This import _should_ appear in the top stanza (alongside `logging` since they're both part of the standard library, but alphabetized).

This change should be part of the initial commit where we move the class definition, so the easiest way to address that is to:
- add a new commit that puts in in the right place
- rebase to reorder and squash the new commit into the first
- resolve any conflicts by keeping the `import` in the proper location.